TicTacToe2/TicTacToeGame.cs

```diff
@@ -6,15 +6,14 @@ namespace TicTacToe2
 {
     class TicTacToeGame
     {
-        static readonly string[] space = new string[9] {"1", "2", "3", "4", "5", "6", "7", "8", "9"};
+        private readonly string[] space = new string[9] { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
         public void Play()
         {
             bool winner = false;
             bool draw = false;
             while (!winner && !draw)
             {
                 DrawBoard();
-                Console.ReadLine();
```

Comment:
Console.ReadLine(); removed. I was using it as a place holder to read the screen when I ran the debugger. No longer needed since I started creating GetUserInput().